examples/web/README.md

```diff
@@ -0,0 +1,15 @@
+# Lens SDK + wagmi + Bundlr
```

Comment:
maybe note here that the example is using `ethers` as well

people may be looking for a `viem` example so being explicit about which tool here would be useful